### DISCUSSION TITLE
Access environment variable

### DISCUSSION THREAD

**Discussion by sujanshsh** [quasarframework/quasar#15221] [CLI - PWA mode] [answered]
In nuxt js, process.env.SOME_VARIABLE gets replaced by its value at build time. How can we access environment variable in quasar app when deployed in nginx?

**Comment by suleimanas** [comment, marked as answer]
same:
https://quasar.dev/quasar-cli-webpack/handling-process-env
https://quasar.dev/quasar-cli-vite/handling-process-env

note:
you are building PWA so it'll be replaced at build

**Reply by sujanshsh** [reply]
thank you
